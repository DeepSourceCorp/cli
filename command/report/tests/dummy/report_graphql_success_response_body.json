{
  "data": {
    "createArtifact": {
      "ok": true,
      "error": ""
    }
  }
}
