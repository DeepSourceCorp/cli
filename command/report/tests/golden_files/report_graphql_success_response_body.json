{
  "data": {
    "createArtifact": {
      "ok": true,
      "error": "",
      "message": "Artifact successfully uploaded for repository demo-python on commit d3401b650b2b3f1c2b4008dfd934d00bf3590b78."
    }
  }
}
